trade-finance-manager-api/src/v1/api.js

```diff
@@ -164,6 +164,10 @@ const updatePortalFacility = async (facilityId, update, auditDetails) => {
   }
 };
 
+/**
+ * @param {string} dealId
+ * @returns {Promise<import('@ukef/dtfs2-common').TfmDeal}
```

Comment:
This isn't _strictly_ accurate since the database models will be transformed when converted to JSON. Should I add a new DTO to reflect this?

Reply:
What's the transformation? Might be overkill for `api.js`

Reply:
In general, everything that's sent as JSON will lose some information - e.g. `Date` and `ObjectId` properties will be converted to string. We could get around this by adding a `Jsonify` type helper (a la https://effectivetypescript.com/2020/04/09/jsonify/) 

for the deal in particular `dtfs-central-api\src\v1\controllers\tfm\deal\tfm-get-deal.controller.js` does some weird mapping, though for my particular use case it doesnt have any impact.

Keeping _as-is_ is the simplest thing to do but it could cause confusion down the line. I'd caution against trusting any of the types declared in a `js` file anyway, so its not tooo big an issue

Reply:
Agree @MarRobSoftwire 

I think it would be overkill at this point in time to be super specific in `api.js`, especially considering it's essentially a "middleman".